src/instr/instr_smpi.h

```diff
@@ -29,7 +29,6 @@ XBT_PRIVATE void TRACE_smpi_testing_out(int rank);
 XBT_PRIVATE void TRACE_smpi_testing_in(int rank, instr_extra_data extra);
 XBT_PRIVATE void TRACE_smpi_alloc();
 XBT_PRIVATE void TRACE_smpi_release();
-XBT_PRIVATE void TRACE_smpi_ptp_in(int rank, const char* operation, instr_extra_data extra);
```

Comment:
This is a completely unrelated change, so don't put this in the same pull request.

Moreover, I already said in the previous pull request that this change was not right. I don't want to merge the ptp_in signal with the collective_in signal.